main.go

```diff
@@ -25,34 +38,36 @@ func main() {
 		panic(err.Error())
 	}
 
+	fmt.Println("Beginning the crawl.")
+
 	for {
 		pods, err := clientset.CoreV1().Pods("").List(metav1.ListOptions{})
 		if err != nil {
 			panic(err.Error())
 		}
 
-		restartThreshold := int32(100)
+		restartThreshold := int32(NUMBER_RESTARTS)
 
 		for _, item := range pods.Items {
 			for _, status := range item.Status.ContainerStatuses {
 				waiting := status.State.Waiting
 
 				if waiting != nil {
 					reason := waiting.Reason
-					if reason == "CrashLoopBackOff" && status.RestartCount > restartThreshold {
+					if stringInSlice(reason, waitingReasons) && status.RestartCount > restartThreshold {
```

Comment:
@jonathantronson Here is an idea I had for choosing if the waiting reason is in a slice of possibilities. What do you think?

Reply:
https://stackoverflow.com/a/10486196/6520477

I like this idea. I went looking for an "includes" or "contains" type of method, and came up with that SO question. It's probably more efficient to do it the way they discuss in the SO question. Since this loop touches every pod, we might like that efficiency.